plugins/tekton/src/components/common/ClusterSelector.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { TektonResourcesContext } from '../../hooks/TektonResourcesContext';
 import { Select, SelectedItems } from '@backstage/core-components';
 
-const ClusterSelector: React.FC = () => {
+export const ClusterSelector: React.FC = () => {
```

Comment:
we can use default export, if you prefer named export and good to remove default one